### PR TITLE
refactor: move several sources as built-in library

### PR DESCRIPTION
**Pull request by fpistm**
Several reasons:
- Avoid licenses conflict
- Improve build time on windows
- 1 feature per library

### CODE REVIEW

File: License.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ Note: most license information is available on top of each source file
 
 * cores/arduino/stm32/ mainly contains source from STMicroelectronics.
 * system/Drivers/STM32*xx_HAL_Driver folders include the STMicroelectronics HAL Drivers.
+* system/Middlewares/OpenAMP folders.
```
Comment (mathieuchopstm) [inline License.md:13]: Nit:
```suggestion
* system/Middlewares/OpenAMP/
```

File: libraries/VirtIO/src/VirtIO.h
Comment (mathieuchopstm) [inline libraries/VirtIO/src/VirtIO.h:1]: `License.md` is a bit unclear about which license this is under, maybe adding MIT License header would resolve this confusion.

File: libraries/USBDevice/examples/BareMinimum/BareMinimum.ino
Comment (mathieuchopstm) [inline libraries/USBDevice/examples/BareMinimum/BareMinimum.ino:1]: Is this file necessary?

File: libraries/USBDevice/library.properties [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name=USBDevice
+version=1.0.0
+author=Frederic Pillon
+maintainer=stm32duino
+sentence=Enables the USB device support CDC or HID.
```
Comment (mathieuchopstm) [inline libraries/USBDevice/library.properties:5]: Nit:
```suggestion
sentence=Enables USB device support (CDC or HID).
```

File: libraries/VirtIO/library.properties [hunk 3]
```diff
@@ -0,0 +1,9 @@
+name=VirtIO
+version=1.0.0
+author=Bumsik kim
+maintainer=stm32duino
+sentence=Enables the communication between cortex-A7 (Linux) and cortex-m4 (Arduino) via shared memory.
```
Comment (mathieuchopstm) [inline libraries/VirtIO/library.properties:5]: Nit:
```suggestion
sentence=Enables serial communication between Cortex-A7 (Linux) and Cortex-M4 (Arduino) on STM32 MPUs via shared memory.
```

**Review by mathieuchopstm — COMMENTED** [review]
Looks good!
A few comments:

File: libraries/USBDevice/examples/BareMinimum/BareMinimum.ino
Comment (fpistm) [reply libraries/USBDevice/examples/BareMinimum/BareMinimum.ino:1]: yes for arduino-lint :)
